tests/integration/utils_test.py

```diff
@@ -73,7 +73,7 @@ def create_trip_update(vj_id, trip_id, circulation_date, contributor_id):
 
 
 def create_rt_update_and_trip_update(id, contributor_id, connector_type, vj_id, trip_id, circulation_date):
```

Comment:
:mushroom: there too
```suggestion
def create_rt_update_and_trip_update(id, contributor_id, vj_id, trip_id, circulation_date):
```

Reply:
Ahhh, I saw it... but seems like I forgot to do it :shrug: Done in b1fb44a.